### PR TITLE
New version: YO3DMU.PstRotatorAz version 14.90

### PR DESCRIPTION
**Pull request by Jvr2022**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155689)

### CODE REVIEW

**Comment by stephengillie** [comment]
URL: https://pstrotator.com/ANT/PstRotatorAz.exe 
OK

(Automated message - build 899)

**Comment by stephengillie** [comment]
Hi @Jvr2022,

Is this project moving domains? The `PackageUrl` still points at the old page. Also, neither page references the other. I couldn't find a developer blog or anywhere mentioning a site move - is there any?

**Comment by Jvr2022** [comment]
I am gonna take a look

**Comment by Jvr2022** [comment]
If I go to this link I get the download https://pstrotator.com/ANT/PstRotatorAz.exe

File: manifests/y/YO3DMU/PstRotatorAz/14.90/YO3DMU.PstRotatorAz.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: YO3DMU.PstRotatorAz
+PackageVersion: '14.90'
+InstallerType: inno
+ReleaseDate: 2024-02-15
+Installers:
+- Architecture: x86
+  InstallerUrl: https://pstrotator.com/ANT/PstRotatorAz.exe
```
Comment (stephengillie) [inline manifests/y/YO3DMU/PstRotatorAz/14.90/YO3DMU.PstRotatorAz.installer.yaml:10]:
```suggestion
  InstallerUrl: https://www.qsl.net/yo3dmu/ANT/PstRotatorAz.exe
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Should this `InstallerUrl` be used instead? It comes from the `PackageUrl`.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**